modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-about-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= About backing up data from one cluster and restoring it on another cluster
+
+{oadp-first} is designed to back up and restore application data in the same {product-title} cluster. {mtc-full} ({mtc-short}) is designed to migrate containers, including application data, from one {product-title} cluster to another.
```

Comment:
All other uses of this phrase state _to another cluster_. Probably should be consistent throughout.